mmv1/provider/terraform/custom_code.rb

```diff
@@ -121,6 +121,9 @@ class CustomCode < Api::Object
       # resource was successfully deleted. Use this if the API responds
       # with a success HTTP code for deleted resources
       attr_reader :test_check_destroy
+      # This code is run before the Read call happens.  It's placed
+      # in the Read function.
+      attr_reader :pre_read
```

Comment:
super nit: both here and in validate, can you put this between the last create and first update override so they're in CRUD order?

Reply:
Fixed in new PR